### PR TITLE
Clarify what NULL does in cagg APIs

### PR DESCRIPTION
**Pull request by charislam**
# Description

Clarify the behavior of `NULL` in `add_continuous_aggregate_policy`. Warn that `NULL` as the end offset is not recommended.

# Version

Which documentation version does this PR apply to?

- [x] Latest (Default)
- [ ] Version 1.7
- [ ] Older [specify]

# Links

Partial #910

### CODE REVIEW

**Review by Loquacity — APPROVED** [review]

File: api/refresh_continuous_aggregate.md [hunk 1]
```diff
@@ -26,10 +26,17 @@ refreshing in order to align with bucket boundaries.
...
-|---|---|---|
-| `continuous_aggregate` | REGCLASS | The continuous aggregate to refresh. |
-| `window_start` | INTERVAL | Start of the window to refresh, has to be before `window_end`. `NULL` is eqivalent to `MIN(timestamp)` of the hypertable. |
-| `window_end` | INTERVAL | End of the window to refresh, has to be after `window_start`. `NULL` is eqivalent to `MAX(timestamp)` of the hypertable. |
+|-|-|-|
+|`continuous_aggregate`|REGCLASS|The continuous aggregate to refresh.|
+|`window_start`|INTERVAL|Start of the window to refresh, has to be before `window_end`. `NULL` is eqivalent to `MIN(timestamp)` of the hypertable.|
+|`window_end`|INTERVAL|End of the window to refresh, has to be after `window_start`. `NULL` is eqivalent to `MAX(timestamp)` of the hypertable.|
+
+<highlight type="warning">
```
Comment (gayyappan) [inline api/refresh_continuous_aggregate.md:34]: This warning applies only to the policy. It is not needed for the refresh_continuous_aggregate case. There are cases where it is valid to call this function with NULL for window_end. Could you remove this warning from this function?

File: api/refresh_continuous_aggregate.md [hunk 2]
```diff
@@ -26,10 +26,17 @@ refreshing in order to align with bucket boundaries.
...
+While setting `window_end` to `NULL` is possible, it is not recommended. To 
+include the most recent data in your aggregates, use 
+[real-time aggregation](/timescaledb/latest/how-to-guides/continuous-aggregates/real-time-aggregates/)
+instead.
+</highlight>
```
Comment (charislam) [inline api/refresh_continuous_aggregate.md:40]:
```suggestion
```

File: api/refresh_continuous_aggregate.md (hunk 1 above)
Comment (charislam) [reply api/refresh_continuous_aggregate.md]: Done! Is it good otherwise, or are there other changes I should make?

**Review by gayyappan — APPROVED** [review]
LGTM

**Pull request merged**